chompfile.toml

```diff
@@ -93,3 +94,8 @@ custom-css = 'typedoc/typedoc-theme.css'
 plugins = './typedoc/typedoc-theme.cjs,typedoc-plugin-versions,typedoc-plugin-external-resolver'
 flags = '--lightHighlightTheme $TYPEDOC_SYNTAX_THEME --darkHighlightTheme $TYPEDOC_SYNTAX_THEME'
 out = 'public_html/docs/jspm-cli'
+
+[[task]]
+name = 'generate:api:css'
+run = 'tailwindcss -i ./public_html/styles/input.css -o ./public_html/style.main.css'
+invalidation = 'always'
```

Comment:
To integrate, `public_html/style.main.css` should be added into the dependencies of the top-level tasks.

The `dep` is then `public_html/styles/input.css`. Then invalidation doesn't need to be set and will happen automatically.

Also can we move `public_html/styles/input.css` outside of public_html if it doesn't need to be served itself?